docs/botkube-cloud-auth.md

```diff
@@ -0,0 +1,40 @@
+---
+id: botkube-cloud-auth
+title: Authentication with Botkube Cloud
+sidebar_position: 8
+---
+
+# Authentication with Botkube Cloud
```

Comment:
In general, the idea of providing information about what you gain by having a cloud account (effectively login) is great. Nesting that under "Auth with Botkube" doesn’t convince me much. I’d see it more as a general "comparison" where you have "not logged in, logged in, paid" users. Maybe something similar to the ArgoCD approach - with diagrams, we could also showcase the OOTB team features.

<img width="1295" alt="Screenshot 2024-10-07 at 12 06 23" src="https://github.com/user-attachments/assets/4f4ae8f1-7fc7-42b9-a7ce-b7ac4aca5dd2">

Anyway, it’s probably something for later, so feel free to merge it as it is 👌